README.md

```diff
@@ -1 +1,49 @@
-# bubble-sort-project
\ No newline at end of file
+# bubble-sort-project
+In this project was posible the development of the bubble sort algorithm (to sort an array in ascending order). The assignment is focused on the use of yield and passing a block to a method. All of this is built in Ruby.
+
+![screenshot](https://github.com/Huemac-Alfredo/bubble-sort-project/blob/feature/screen_shot.PNG)
+
+## Built With
+
+- Ruby
+- Rubocop
+- Visual Studio Code
+
+## Live Demo
+
+[Live Demo Link](https://repl.it/@huemacAlfredo/Bubblesort#main.rb)
+
+### Prerequisites
+
+- Browser (Google Chrome, Mozilla Firefox, Safari or any other browser)
+
+- Install: Clone the repo into your local machine and 
+
+- Usage: open de bubble_sort.rb file with your preffered text editor (VS Code is suggested)
+
+## Authors
+
+👤 **Alfredo Huemac**
+
+- Github: [@githubhandle](https://github.com/Huemac-Alfredo)
+- Twitter: [@twitterhandle](https://twitter.com/AlfredoHuemac)
+- Linkedin: [linkedin](https://www.linkedin.com/in/alfredo-huemac-c%C3%B3rdova-173b481b2/)
```

Comment:
Instead of using those default texts on your links `github handle` or `twitter handle`, use your username that you use on that specific social link